packages/sit-onyx/src/components/OnyxListbox/OnyxListbox.vue

```diff
@@ -52,85 +58,133 @@ const slots = defineSlots<{
 
 const { t } = injectI18n();
 
+const isExpanded = ref(false);
+
+watchEffect(async () => {
+  if (isExpanded.value) {
+    await nextTick();
+    miniSearch.value?.focus();
+  }
+});
+
 /**
  * Currently (visually) active option.
  */
-const activeOption = ref<TValue>();
+const activeValue = ref<TValue>();
 
-const searchInput = computed({
-  get: () => (props.withSearch && props.searchTerm) || "",
-  set: (newValue) => emit("update:searchTerm", newValue),
+const arrayValue = computed(() => {
+  if (!props.modelValue) {
+    return [];
+  }
+  return props.multiple ? props.modelValue : [props.modelValue];
 });
```

Comment:
```suggestion
const arrayValue = computed(() => {
  if (!props.modelValue) return [];
  return props.multiple && Array.isArray(props.modelValue) ? props.modelValue : [props.modelValue];
});
```

The potential error that you showed me earlier when playing around with the Storybook controls could maybe be fixed with this minor change